django_cloud_deploy/skeleton/templates/admin_template/admin.py-tpl

```diff
@@ -1 +1,69 @@
-#Custom Admin Code
\ No newline at end of file
+import os
+
+from django.contrib import admin
+from django.template.response import TemplateResponse
+from django.urls import path
+from google.cloud import logging
+from google.oauth2 import service_account
+
+
+
+class CloudAdminSite(admin.AdminSite):
+  def get_urls(self):
+    urls = super().get_urls()
+    my_urls = [
+      path('cloud/',  self.admin_view(self.cloud_view)),
+    ]
+    return my_urls + urls
+
+  def cloud_view(self, request):
+    logs = self.get_logs()
+    context = dict(
+        # Include common variables for rendering the admin template.
+        self.each_context(request),
+        # Anything else you want in the context...
+        logs=logs
+    )
+    return TemplateResponse(request,
+                            "cloud_admin/cloud.html",
+                            context)
+
+  def get_logs(self):
+    if logging_client is None:
+        return ['Check console for logs.']
+
+    query_filter = (
+      ['resource.type="container"',
+       'resource.labels.cluster_name="{{ project_name }}"',
+       'resource.labels.namespace_id="default"',
+       'resource.labels.project_id="{{ project_id }}"',
+       'resource.labels.zone:"us-west1-a"',
+       'resource.labels.container_name="{{ project_name }}-app"'])
+
+    query_filter = ' AND '.join(query_filter)
+    from google.cloud.logging import DESCENDING
+
+    # Show ten entries
```

Comment:
Will we show more than 10 logs in the future?

Reply:
maybe, from what I understand this is more of a POC regarding having more integration with GCP.